pkg/printer/printer.go

```diff
@@ -78,6 +80,8 @@ func New(fopts ...Option) *Printer {
 const (
 	tab     = "\t"
 	newline = "\n"
+
+	nodeStatusCutoff = 10
```

Comment:
I think that `10` is already too many as nodes typically have long names (eg `ip-192-168-12-253.us-west-2.compute.internal`) in practice. What about `2` or `3` instead? Or even better: display as many names as you can given a fixed string length.